deploy/crds/common/monitoring.rhobs_monitoringstacks.yaml

```diff
@@ -56,9 +56,10 @@ spec:
                 - error
                 type: string
               namespaceSelector:
-                description: Namespace selector for Monitoring Stack Resources. If
-                  left empty the Monitoring Stack will only match resources in the
-                  namespace it was created in.
+                description: 'Namespace selector for Monitoring Stack Resources. Set
```

Comment:
@sthaha Not sure why the `single quote` is added after description

Reply:
I don't think this matters as long as the string is valid. (and the test pass)